integrations/sparse/test_ltr_msmarco_document.py

```diff
@@ -32,33 +29,53 @@ def test_reranking(self):
         inp = 'run.msmarco-pass-doc.bm25.txt'
         outp = 'run.ltr.msmarco-pass-doc.test.trec'
         outp_tsv = 'run.ltr.msmarco-pass-doc.test.tsv'
-        #Download prebuilt index
-        #retrieve candidate
+        # Download prebuilt index
```

Comment:
@stephaniewhoo Should we download the index somewhere or change the index below into pre-built index name?

Reply:
use pre-built.